contracts/DaoModule.sol

```diff
@@ -71,17 +86,20 @@ contract DaoModule {
         questionArbitrator = address(_executor);
         minimumBond = bond;
         template = templateId;
-    }
 
-    modifier executorOnly() {
-        require(msg.sender == address(executor), "Not authorized");
-        _;
+        if (_executor != address(0)) {
+            __Ownable_init();
+            transferOwnership(_owner);
+            initialized = true;
```

Comment:
Why do we allow to call this method multiple time? Shouldn't we just enforce that the `_executor` is set on the first (and only) call? 

If there is a good reason we should comment it